yarn.lock

```diff
@@ -16,6 +16,24 @@
     call-me-maybe "^1.0.1"
     js-yaml "^3.13.1"
 
+"@apollo/client@^3.0.2":
```

Comment:
Huh?

Reply:
I think it's just a nested dep of `graphql-tools`